### PR TITLE
feat(graphics): Add a brown haze to systems that are in the center of the dust nebula

### PR DESCRIPTION
**Pull request by RisingLeaf**
**Content (Artwork)**

## Summary
With the new map sprite ( at least I think it comes with the new one), there is some sort of nebula that extends in a curve from talita and ends somewhere in front of gegno space.
Screenshot 2022-12-27 at 8 55 34 PM

This PR adds a brown haze to all systems that are in the center of this nebula, just to add some spice to the game (I always have great fun finding a system that is different.

Because I do want to keep it something that is not very common, only the following systems have this haze:

- Almaaz
- Chirr'ay'akai
- "Ehma Ti"
- Kiru'kichi
- Mebsuta
- Mirzam
- Unagi

## Screenshot
Screenshot 2022-12-28 at 1 05 51 PM

## Save File
Just go to the systems.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] The Image I added is the asset as it just a hue adapted version of the standard haze, the original has also no asset

### CODE REVIEW

**Comment by RisingLeaf** [comment]
screenshot added

**Comment by MasterOfGrey** [comment]
Nice.

You'll also want to add it to the "Al Hurr" system which is in `Hai Reveal 6.txt` which is also put smack in that Nebula.

**Review by EjoThims — APPROVED** [review]

**Comment by Saugia** [comment]
Agree with Azure. If we want to add new bg haze because of the sprite, use the same method for regular haze. EW, etc. haze is unique and only intended for their respective areas.

**Comment by Quantumshark** [comment]
I agree that this needs to be more distinct from the Ember Waste haze.
I am red-green colourblind, and to me this looks almost identical to the red haze in the Ember Waste.

**Comment by beccabunny** [comment]
An option could the the one of using multiple hazes - for instance, both the standard blue haze and this new one - together for a less EW-like effect, by putting them as frames of the same animation but without specifying anything else in the data. E.g. haze-brown+0 being this and haze-brown+1 being the standard blue haze

**Comment by RisingLeaf** [comment]
Using multiple hazes now

**Comment by RisingLeaf** [comment]
> Nice.
> 
> You'll also want to add it to the "Al Hurr" system which is in `Hai Reveal 6.txt` which is also put smack in that Nebula.

is in

**Comment by RisingLeaf** [comment]
also it is adpated from the standard haze now

File: copyright [hunk 1]
```diff
@@ -1961,6 +1961,12 @@ Files:
 Copyright: Zitchas (zitchas.jma@gmail.com)
 License: CC-BY-SA-4.0
 
+Files:
+ images/_menu/haze-brown*
+Copyright: RisingLeaf (https://github.com/RisingLeaf)
+License: CC-BY-SA-4.0
+Comment: Created on base of haze-133 (no copyright given).
```
Comment (Saugia) [inline copyright:1968]: Just making it a bit cleaner.

```suggestion
Comment: Derived from _menu/haze-133 (no copyright given).
```

**Review by Saugia — CHANGES_REQUESTED** [review]

**Review by Saugia — APPROVED** [review]

**Pull request merged**